duckduckgo/Android
# Fire education - release

Task/Issue URL: https://app.asana.com/0/488551667048375/1199361818043729/f
Tech Design URL: 
CC: 

**Description**:
This PR introduces necessary changes to release Fire Button education to all users. We have also introduced some changes coming from our last product review.

See notes in https://app.asana.com/0/488551667048375/1199361818043729/f

**Steps to test this PR**:
> Common scenario
1. Fresh install
1. Visit any site with trackers
1. Dismiss Dax CTA
1. Ensure Fire pulsing animation is active
1. Click on the Fire button
1. Ensure CTA appears on top of clear data options

> User opens more than 1 tabs
1. Fresh install
1. perform a search
1. Dismiss Dax CTA
1. open a new tab
1. Visit a site with trackers
1. Dismiss CTA
1. Ensure Fire pulsing animation does not appear
1. Click on the Fire button
1. Ensure CTA does not appear

> User opens more than 1 tabs after pulsing animation started
1. Fresh install
1. Visit any site with trackers
1. Dismiss Dax CTA
1. Ensure Fire pulsing animation is active
1. Open a new tab
1. Ensure pulsing animation is not active
1. Click on the Fire button
1. Ensure CTA does not appear

> Database migration
1. Install develop
1. Start the onboarding flow
1. Install this branch
1. Open the app, ensure onboarding was completed

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
